src/components/Common/RequirementsSelector.tsx

```diff
@@ -124,6 +124,15 @@ export default function RequirementsSelector({
   const [isOpen, setIsOpen] = React.useState(false);
   const [isCreateSheetOpen, setIsCreateSheetOpen] = React.useState(false);
 
+  const onSuccess = () => {
+    setIsCreateSheetOpen(false);
+    setIsOpen(false);
```

Comment:
in this case, let's not add this; user should be able to create multiple items without having to reopen the RequirementsSelector Sheet.